shared/chat/conversation/messages/message-popup/attachment.tsx

```diff
@@ -18,24 +18,20 @@ const emptyMessage = C.Chat.makeMessageAttachment({})
 
 const PopAttach = (ownProps: OwnProps) => {
   const {ordinal, attachTo, onHidden, position, style, visible} = ownProps
-  const m = C.useChatContext(s => s.messageMap.get(ordinal))
+  const m = C.useChatContext(s => s.messageMap.get(ordinal) ?? s.messageMapAttachments.get(ordinal))
```

Comment:
things that are launched from the info panel can be only in the fallback so always look there